### PR TITLE
Added a short summary and "why bother"

### PR DESCRIPTION
**Pull request by Nyholm**

### CODE REVIEW

File: proposed/http-client/http-client-meta.md [hunk 1]
```diff
@@ -3,8 +3,12 @@ HTTP Client Meta Document
 
 ## Summary
 
+HTTP request and responses are the two fundamental objects in web programming. All clients communicating to an external API is using some form of HTTP client. Many libraries are coupled to one specific client or is implementing a client themselves. This leads to bad library design, version conflicts or too much code note related to the library domain.
```
Comment (sagikazarmark) [inline proposed/http-client/http-client-meta.md]: All clients communicating to an external API use some form of HTTP client

Comment (sagikazarmark) [inline proposed/http-client/http-client-meta.md]: Many libraries are coupled to one specific client or implement a client and/or adapter layer themselves.

Comment (sagikazarmark) [inline proposed/http-client/http-client-meta.md]: not~~e~~ related

File: proposed/http-client/http-client-meta.md [hunk 2]
```diff
@@ -3,8 +3,12 @@ HTTP Client Meta Document
 
 ## Summary
 
+HTTP request and responses are the two fundamental objects in web programming. All clients communicating to an external API is using some form of HTTP client. Many libraries are coupled to one specific client or is implementing a client themselves. This leads to bad library design, version conflicts or too much code note related to the library domain. 
+
 ## Why bother?
 
+With PSR-7 we know how a HTTP request and response look like. But there is nothing that defines how a request should be sent. A common interface for HTTP client will allow libraries to be decoupled from HTTP client implementation such as Guzzle.
```
Comment (sagikazarmark) [inline proposed/http-client/http-client-meta.md]: Thanks to PSR-7 ...... ideally(?) look like.

Comment (sagikazarmark) [inline proposed/http-client/http-client-meta.md]: But nothing defines

**Comment by Nyholm** [comment]
Thank you for the review

File: proposed/http-client/http-client-meta.md (hunk 2 above)
Comment (sagikazarmark) [inline proposed/http-client/http-client-meta.md]: how a request should be sent and a response received?

**Comment by sagikazarmark** [comment]
Some comments were marked as outdated too early..

File: proposed/http-client/http-client-meta.md (hunk 2 above)
Comment (sagikazarmark) [reply proposed/http-client/http-client-meta.md]: be decoupled from an HTTP client implementation such as Guzzle.

File: proposed/http-client/http-client-meta.md [hunk 3]
```diff
@@ -3,8 +3,19 @@ HTTP Client Meta Document
 
 ## Summary
 
+HTTP request and responses are the two fundamental objects in web programming. 
+All clients communicating to an external API use some form of HTTP client. Many
+libraries are coupled to one specific client or implement a client and/or adapter
+layer themselves. This leads to bad library design, version conflicts or too much
+code note related to the library domain.
```
Comment (sagikazarmark) [inline proposed/http-client/http-client-meta.md]: NOT related

**Comment by Nyholm** [comment]
Yeah, I saw that. I've updated the PR again

File: proposed/http-client/http-client-meta.md [hunk 4]
```diff
@@ -3,8 +3,19 @@ HTTP Client Meta Document
...
 
+HTTP request and responses are the two fundamental objects in web programming. 
+All clients communicating to an external API use some form of HTTP client. Many
+libraries are coupled to one specific client or implement a client and/or adapter
+layer themselves. This leads to bad library design, version conflicts or too much
+code not related to the library domain. 
+
 ## Why bother?
 
+Thanks to  PSR-7 we know how a HTTP request and response ideally look like. But nothing
```
Comment (sagikazarmark) [inline proposed/http-client/http-client-meta.md]: > Thanks to  PSR-7

Double space

Comment (sagikazarmark) [inline proposed/http-client/http-client-meta.md]: how HTTP requests and responses

Comment (sagikazarmark) [inline proposed/http-client/http-client-meta.md]: But nothing... Could be the same sentence as the one before it? , but nothing..

File: proposed/http-client/http-client-meta.md [hunk 5]
```diff
@@ -3,8 +3,19 @@ HTTP Client Meta Document
...
+All clients communicating to an external API use some form of HTTP client. Many
+libraries are coupled to one specific client or implement a client and/or adapter
+layer themselves. This leads to bad library design, version conflicts or too much
+code not related to the library domain. 
+
 ## Why bother?
 
+Thanks to  PSR-7 we know how a HTTP request and response ideally look like. But nothing 
+defines how a request should be sent and a response received. A common interface for HTTP
+client will allow libraries to be decoupled from an HTTP client implementation such as
```
Comment (sagikazarmark) [inline proposed/http-client/http-client-meta.md]: to be decoupled from an implementation (HTTP Client twice)

**Comment by Nyholm** [comment]
I'll merge this. If there is more input, feel free to send a new PR.

**Pull request merged**